packages/bee-q/src/components/breadcrumb-item/bq-breadcrumb-item.tsx

```diff
@@ -0,0 +1,133 @@
+import { h, Component, Prop, Element, Event, EventEmitter } from '@stencil/core';
+
+import { isDefined } from '../../shared/utils';
+
+@Component({
+  tag: 'bq-breadcrumb-item',
+  styleUrl: './scss/bq-breadcrumb-item.scss',
+  shadow: true,
+})
+export class BqBreadcrumbItem {
+  // Own Properties
+  // ====================
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLBqBreadcrumbItemElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  /** If true, the item is the last element inside breadcrumb */
+  @Prop() hideSeparatorIcon: boolean = false;
+
+  /** The icon name used as separator. Default is `/`. */
+  @Prop() separatorIcon!: string;
+
+  /** If set, the breadcrumb item will be rendered as an `<a>` with this `href`, otherwise, a `<button>` will be rendered. */
+  @Prop({ reflect: true }) href: string;
+
+  /** Where to display the link in the browser context. Relevant only if `href` is set. */
+  @Prop({ reflect: true }) target: '_blank' | '_parent' | '_self' | '_top';
+
+  /** Where to display the link in the browser context. Relevant only if `href` is set. */
+  @Prop({ reflect: true }) rel: string = 'noreferrer noopener';
+
+  // Prop lifecycle events
+  // =======================
+
+  // Events section
+  // Requires JSDocs for public API documentation
+  // ==============================================
+
+  /** Handler to be called when item loses focus */
+  @Event() bqBlur: EventEmitter<HTMLBqBreadcrumbItemElement>;
+
+  /** Handler to be called when item is focused */
+  @Event() bqFocus: EventEmitter<HTMLBqBreadcrumbItemElement>;
+
+  /** Handler to be called when item is clicked */
+  @Event() bqClick: EventEmitter<HTMLBqBreadcrumbItemElement>;
+
+  // Component lifecycle events
+  // Ordered by their natural call order
+  // =====================================
+
+  // Listeners
+  // ==============
+
+  // Public methods API
+  // These methods are exposed on the host element.
+  // Always use two lines.
+  // Public Methods must be async.
+  // Requires JSDocs for public API documentation.
+  // ===============================================
+
+  // Local methods
+  // Internal business logic.
+  // These methods cannot be called from the host element.
+  // =======================================================
+
+  private onBlur = () => {
+    this.bqBlur.emit(this.el);
+  };
+
+  private onFocus = () => {
+    this.bqFocus.emit(this.el);
+  };
+
+  private onClick = () => {
+    this.bqClick.emit(this.el);
+  };
+
+  // render() function
+  // Always the last one in the class.
+  // ===================================
+
+  render() {
+    const isLink = isDefined(this.href);
+    const TagElem = isLink ? 'a' : 'button';
```

Comment:
I don't think we need a `button` for the breadcrumb navigation, the acts as links that navigate users to a page, right?

Reply:
Yes, default tag element for the items is `button`. But, if `href` is provided, then we use anchor tag for navigation to a different path or to a different section within the same path.
Maybe we should remove the `button` and only use the `a` element. And if `href` is not provided then use `span`? (maybe the consumer only wants to show a read-only breadcrumb)